app/models/dfe_subject.rb

```diff
@@ -1,40 +1,46 @@
 class DFESubject
   FINANCIAL_SUPPORT_FOR_2019_20_RECRUITMENT_CYCLE = [
-    { subjects: %w[Mathematics], bursary_amount: 20000, early_career_payments: 10000, scholarship: 22000 },
-    { subjects: %w[French Chemistry Computing Spanish Geography Physics German], bursary_amount: 26000, early_career_payments: nil, scholarship: 28000 },
-    { subjects: ["Modern languages (other)", "Classics", "Italian", "Japanese", "Mandarin", "Russian", "Biology"], bursary_amount: 26000, early_career_payments: nil, scholarship: nil },
-    { subjects: %w[English], bursary_amount: 15000, early_career_payments: nil, scholarship: nil },
-    { subjects: ["Design and technology", "History"], bursary_amount: 12000, early_career_payments: nil, scholarship: nil },
-    { subjects: %w[Music], bursary_amount: 9000, early_career_payments: nil, scholarship: nil },
-    { subjects: ["Primary with mathematics"], bursary_amount: 6000, early_career_payments: nil, scholarship: nil },
+    { subjects: %w[Mathematics], bursary_amount: 20000, early_career_payments: 10000, scholarship: 22000, precedence: 7 },
```

Comment:
precedence potential issues
```ruby

{ subjects: %w[new_subject_2], bursary_amount: 10, scholarship: 0,         precedence: 9 }
{ subjects: %w[new_subject_1], bursary_amount: 0,   scholarship: 22000, precedence: 8 }
```

Given  course with subjects
```ruby

course1.subject = [Mathematics, new_subject_1 ]
course2.subject = [Mathematics, new_subject_2 ]
course3.subject = [Mathematics, new_subject_1, new_subject_2 ]
```

the `course2.bursary_amount` & `course3.bursary_amount` & `course1.scholarship_amount` looks like it can be zeroed (from `WithFinancialIncentives`)

maybe a comment or another look at this precedence thing?